tests/rptest/tests/rpk_cluster_test.py

```diff
@@ -86,6 +86,8 @@ def test_debug_bundle(self):
                 # dmidecode doesn't work in ducktape containers, ignore
                 # errors about it.
                 continue
+            if 'ntp' in l:
```

Comment:
We should ignore I/O timeouts specifically rather than any ntp issue at all -- something like a regex match on "error querying.*i/o timeout".  

That way if there was something profoundly broken with the ntp client (e.g. it was sending malformed queries or couldn't parse responses) then we would still fail the test.

Reply:
Makes sense, let me change it!